sycl/test-e2e/Basic/launch_queries/max_sub_group_size.cpp

```diff
@@ -0,0 +1,86 @@
+// REQUIRES aspect-subgroup
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+#include <sycl/detail/core.hpp>
+#include <sycl/detail/info_desc_helpers.hpp>
+#include <sycl/kernel.hpp>
+#include <sycl/kernel_bundle.hpp>
+#include <sycl/sycl.hpp>
```

Comment:
You must not include `sycl.hpp` in E2E tests, it is way to heavy. Instead, you should you specific headers you need directly